ppr-ui/src/views/newMhrRegistration/HomeLocation.vue

```diff
@@ -25,20 +25,28 @@
         :class="{ 'border-error-left': validateCivicAddress }"
       />
     </section>
+    <section id="mhr-home-land-ownership-wrapper" class="mt-10">
```

Comment:
I don't have any conditionals for rendering the component, let me know if I am missing that.

Reply:
The question (for Tracey or whomever) would be does this confirmation apply to both STAFF and Manufacturers. 

I think i can guess here that you are safe without a conditional, as we are only targeting STAFF right now and this step in it's whole isn't part of the Manufacturers flow.

Reply:
tldr: your good 👍